sycl/test/regression/atomic_load.cpp

```diff
@@ -0,0 +1,41 @@
+// RUN: %clangxx -fsycl %s -o %t.out -lOpenCL
+// RUN: env SYCL_DEVICE_TYPE=HOST %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+// RUN: %clangxx --sycl -S -x c++ -O0 -emit-llvm %s -o - | FileCheck %s
```

Comment:
I don't see any value in this check.
It's tightly coupled with the implementation in SYCL headers. Passing on accelerator should be enough.
Please, remove.